README.dev.md

```diff
@@ -65,6 +67,20 @@ You can also use the Majestic web interface to run the unit tests in your browse
 npm run test:unit:ui
 ```
 
+### End-to-end tests
+
+We use [Cypress](https://cypress.io) for end-to-end tests. These tests can be found under the `cypress/e2e` folder.
+
+You can start Cypress with
+
+```shell
+npm run cypress:open
+```
+
+Select "E2E Testing", which should be configured already and select the desired browser and click the start button.
+
+A browser will open with the list of tests, which can be explored.
```

Comment:
Can you please add the next steps to be able to run the tests? If you have time, it would be nice to have screenshots.